### DISCUSSION TITLE
Background color

### DISCUSSION THREAD

**Discussion by loop2** [slipx06/sunsynk-power-flow-card#343] [Q&A] [answered]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current Behavior

I can obtain only black and white background

### Expected behaviour

I would change or remove the background color

### Possible Solutions

card_mod ?

### Mode

Both

### Context / Reason

Hi! Great card! There is a way in order to change or remove the backgroud color of the card?

**Comment by slipx06** [comment, marked as answer]
Hi. You can set this using card_mod. Just add the following to your config

```
card_mod:
  style: |
    ha-card {
      --ha-card-background: green;
    }
```

![image](https://github.com/slipx06/sunsynk-power-flow-card/assets/7227275/e7f945a1-2e8e-4a47-a6a1-1d29b3fbfeaa)
